enhancement/composer.py

```
@@ -0,0 +1,474 @@
+    
+    从真相文件系统中选择与当前章节相关的上下文，
+    并编译规则栈供写作使用。
+    """
+    
+    def __init__(self, truth_files_dir: str = "story/truth_files"):
+        self.truth_files_dir = Path(truth_files_dir)
+    
+    def compile(
+        self,
+        intent: ChapterIntent,
+        truth_files: Dict[str, str] = None,
```

Ruff (0.15.10)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)